release notes/v0.36.0.md

```diff
@@ -0,0 +1,108 @@
+}
+
+export function f3() {
+  throw "line 10";
+}
+```
+
+and a k6 test script importing it as such:
+```javascript
+import { f2 } from "./imported.js"
+
+export default function() {
+  f2();
+}
+```
+
+k6 would indeed report the exception at the correct line in `imported.js`, with an output along the lines of:
+```
+ERRO[0000] line 6
+        at f2 (file:///some/path/imported.js:6:2(2))
+        at file:///some/path/loadtest.js:4:2(4)
+        at native  executor=per-vu-iterations scenario=default source=stacktrace
+```
+
+### Ability to abort tests ([#2093](https://github.com/grafana/k6/pull/2093))
+
+Thanks to the contribution of [@gernest](https://github.com/gernest), k6 now has the ability to abort a test. [#2093](https://github.com/grafana/k6/pull/2093) made the `k6/execution` exposes the `test.abort()` function, which allows to immediately abort a k6 load test run with the exit code `108`.
+
+Aborting is possible during initialization:
+```javascript
+import exec from "k6/execution";
+exec.test.abort();
+```
+
+As well as inside the default function:
+```javascript
+import exec from "k6/execution";
+
+export default function() {
+  // Note that you can abort with a specific message too
+  exec.test.abort("this is the reason");
+}
+
+export function teardown() {
+  console.log("teardown will still be called after test.abort()");
+}
+```
+
+Or during the VU initialization:
+```javascript
+import exec from "k6/execution";
+
+// This won't fail on initial parsing of the script, but on VU initialization.
+if (__VU == 1) {
+  exec.test.abort();
+}
+
+export default function() {}
+```
+
+### k6 inspect extended output ([#2279](https://github.com/grafana/k6/pull/2279))
+
+Following [#2279](https://github.com/grafana/k6/pull/2279), the `k6 inspect` command now supports an `--execution-requirements` flag. When used, the command's output will include fields related to the execution requirements, the configuration k6 derived from the execution context, and will also expose the `maxVUs` and `totalDuration` fields.
+
+
+### Forcing HTTP/1 protocol ([#2222](https://github.com/grafana/k6/pull/2222))
+
+Thanks to the work of [@sjordhani22](https://github.com/sjordhani22), [#2222](https://github.com/grafana/k6/pull/2222) made it possible to force k6 to use version 1.1 of the protocol when firing HTTP requests. 
+
+It can be done by setting the `http2client=0` value in the `GODEBUG` environment variable:
+
+```
+GODEBUG=http2client=0 k6 run testscript.js
+```
+
+**Nota Bene**: the usage of the `GODEBUG` variable is considered temporary, and expected to change in the future. If you start using this feature, keep an eye out for potential future changes.
+
+## Bugs Fixed!
+
+* We had a long-standing issue ([#1771](https://github.com/grafana/k6/issues/1771)) making opening large files in the context of a k6 script somewhat unreliable under specific conditions in the past. This has now been addressed by [#2314](https://github.com/grafana/k6/pull/2314).
```

Comment:
This also has a breaking change component where now some scripts that would've worked in the past won't now.

I would argue this should be expaneded explaining the full change - "opening new files is now only possibly in init context" and explaining why open can only be used in the init context and what is trying to do - more info [here](https://github.com/grafana/k6/issues/557#issuecomment-376115100)

Reply:
Good point! Thanks a lot for the pointers too 🙇🏻